src/setup/plugins.ts

```diff
@@ -2,6 +2,7 @@
 import VueBemCn from '@/plugins/vue-bem-cn';
 import resizeEnd from '@/plugins/resize-end';
 import viewport from '@/plugins/viewport';
+import tooltip from '@/plugins/tooltip';
```

Comment:
suggestion: I would recommend that we (for now) only import the directive and component in the related route, so this is not added to the final bundle if not required.

For directive see: https://vuejs.org/guide/reusability/custom-directives.html#custom-directives